setup.py

```diff
@@ -5,7 +5,9 @@
     "pyyaml~=6.0",
     "wasabi",
     "numpy",
-    "cloudpickle>=1.6"
+    "cloudpickle>=1.6",
+    "gym~=0.21",
+    "stable-baselines3~=1.7",
```

Comment:
not sure if it's a good thing that this packages depends on SB3

Reply:
SB3 is used in `push_to_hub.py` so it makes sense to depend on it?

I set up a venv for `huggingface_sb3`, installed it there and then wanted to run the tests. Those tests failed due to missing gym and sb3 dependencies.
Therefore I added them.

Why do you think we should not add the dependency?

Reply:
oh, that's true, but I think @simoninithomas wanted to switch to gymnasium (so a simple dependency on SB3 should work).